opslevel/common.go

```diff
@@ -32,6 +35,36 @@ import (
 
 // var DefaultPredicateDescription = "A condition that should be satisfied."
 
+const (
+	providerIssueUrl = "https://github.com/OpsLevel/terraform-provider-opslevel/issues"
+	providerFile     = "../tests/provider.tf"
```

Comment:
this seems unused?

Reply:
Good catch. It was being used and I missed this in the previous commit